app/views/events/show.html.erb

```diff
@@ -36,8 +36,8 @@
       </thead>
       <tbody>
         <tr>
-          <% if @donations.blank? %>
-            <h4>There is no Donation from a Donor named <%= params[:search] %>.</h4>
+          <% if @donations.blank? && !params[:search].blank? %>
+            <h4>There is no Donation from a Donor named "<%= params[:search] %>".</h4>
```

Comment:
Not sure if I should create a separate method for these conditions or it's ok to leave it like this in the view  :thinking: